keyboards/wolfmarkclub/wm1/rules.mk

```diff
@@ -0,0 +1,37 @@
+# Build Options
+#   change yes to no to disable
+#
+
+# MCU name
+MCU = STM32F103
+
+# GENERIC STM32F103C8T6 board - stm32duino bootloader
```

Comment:
```suggestion
# GENERIC STM32F103C8T6 board - mass storage bootloader
```

Reply:
it's an RCT6, I guess we should do a full pass